pulp_python/app/migrations/0004_DATA_swap_distribution_model.py

```diff
@@ -0,0 +1,77 @@
+# Generated by Django 2.2.19 on 2021-03-19 17:42
+
+from django.db import migrations, models, transaction
+import django.db.models.deletion
+
+
+def migrate_data_from_old_model_to_new_model_up(apps, schema_editor):
+    """ Move objects from PythonDistribution to NewPythonDistribution."""
+    PythonDistribution = apps.get_model('python', 'PythonDistribution')
+    NewPythonDistribution = apps.get_model('python', 'NewPythonDistribution')
+    for python_distribution in PythonDistribution.objects.all():
+        with transaction.atomic():
+            NewPythonDistribution(
+                pulp_id=python_distribution.pulp_id,
+                pulp_created=python_distribution.pulp_created,
+                pulp_last_updated=python_distribution.pulp_last_updated,
+                pulp_type=python_distribution.pulp_type,
+                name=python_distribution.name,
+                base_path=python_distribution.base_path,
+                content_guard=python_distribution.content_guard,
+                remote=python_distribution.remote,
+                publication=python_distribution.publication
+            ).save()
+            python_distribution.delete()
+
+
+def migrate_data_from_old_model_to_new_model_down(apps, schema_editor):
+    """ Move objects from NewPythonDistribution to PythonDistribution."""
+    PythonDistribution = apps.get_model('python', 'PythonDistribution')
+    NewPythonDistribution = apps.get_model('python', 'NewPythonDistribution')
```

Comment:
When migrating down, the operations are done in reverse order correct? So the 3.12 PythonDistribution would have been renamed back to NewPythonDistribution?

Reply:
That's my understanding